media/js/privacy/privacy-day.js

```diff
@@ -31,52 +31,4 @@ $(function() {
         element: $('#button-nav-wrapper')
     });
 
-    // Setup GA tracking for main tabs
-    $('#tips-nav-direct a').on('click', function() {
-        var thisTabName = $(this).attr('href');
-
-        window.dataLayer.push({
-            event: 'privacy-interactions',
-            browserAction: thisTabName.substr(1),
-            interaction: 'Tab Click'
-        });
-    });
-
-    // Setup GA tracking for paragraph and list links
-    $('.tip-column p a, .tip-column li a').attr({
-        'data-element-action': pager.currentPage.id,
-        'data-interaction': 'href',
-        'data-tracking-flag': 'privacy'
-
-    });
-
-    // Setup GA tracking TED video link
-    $('.tip-column p a, .tip-column li a').attr({
-        'data-element-action': pager.currentPage.id,
-        'data-interaction': 'Why Privacy Matters CTA Btn',
-        'data-tracking-flag': 'privacy'
-    });
-
-    // Setup GA tracking for next tab buttons
-    $('.tip-footer .next a').on('click', function(e) {
-        e.preventDefault();
-        pager.nextPageWithAnimation();
```

Comment:
We need to keep these event handlers for this line of code to allow the user to cycle through the different tabs/pages.